components/MenuBar/MenuBar.jsx

```diff
@@ -0,0 +1,76 @@
+import { PropTypes, Component } from 'react'
+
+require('./MenuBar.scss')
+
+class MenuBar extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {mobile: false}
+  }
+
+  componentWillMount() {
+    this.handleResize(this)
+    const handleResize = this.handleResize
+    const thisRef = this
+    this.state.handler = function() { handleResize(thisRef) }
+    window.addEventListener('resize', this.state.handler)
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.state.handler)
+  }
+
+  handleResize(thisRef) {
+    let breakPoint = thisRef.props.mobileBreakPoint
+    if (!breakPoint) {
+      breakPoint = MenuBar.MobileBreakPoint
+    }
+    if (window.innerWidth > breakPoint) {
+      thisRef.setState({mobile: false})
+    } else {
+      thisRef.setState({mobile: true})
+    }
+  }
+
+  render() {
+    let mbClasses = 'MenuBar'
+    let orientation = this.props.orientation
+    if (!orientation || ['horizontal', 'vertical'].indexOf(orientation) === -1) {
+      orientation = 'horizontal'
+    }
+    mbClasses += (' ' + orientation)
+    return (
+      <ul className={mbClasses}>
```

Comment:
Not sure if it's necessary here, but I've found the `classnames` npm package really useful. https://github.com/JedWatson/classnames